pkg/build/controller/controller.go

```diff
@@ -288,8 +288,12 @@ func (bc *BuildDeleteController) HandleBuildDeletion(build *buildapi.Build) erro
 		glog.V(2).Infof("Did not find pod with name %s for Build %s in namespace %s", podName, build.Name, build.Namespace)
 		return nil
 	}
-	if pod.Labels[buildapi.BuildLabel] != build.Name {
-		glog.V(2).Infof("Not deleting pod %s/%s because the build label %s does not match the build name %s", pod.Namespace, podName, pod.Labels[buildapi.BuildLabel], build.Name)
+	if pod.Labels[buildapi.BuildLabel] != build.Name && pod.Labels[buildapi.DeprecatedBuildLabel] != build.Name {
```

Comment:
should probably refactor this into a generic "getBuildLabel" method that returns string,bool(value,wasLabelFound)